### ISSUE TITLE
silu lowering regression in 1.10

### ISSUE THREAD

**Issue by rwightman** [pytorch/xla#3192]
## 🐛 Bug

```
pt-xla-profiler: Op(s) not lowered: aten::silu_backward.grad_input,  Please open a GitHub issue with the above op lowering requests.
pt-xla-profiler: TransferFromServerTime too frequent: 52 counts during 1 steps
```

This was not a problem in the previous release.

## To Reproduce

Train a model using silu with PyTorch XLA 1.10

## Environment

 - Reproducible on XLA backend [CPU/TPU]: TPU
 - torch_xla version: 1.10

**Comment by JackCaoG** [comment]
I am working on the lowering but I found one issue, Pytorch does not expose `silu_backward` aten string in https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/core/aten_interned_strings.h.

I am planning to backport the lowering to 1.10 to solve Ross's issue but I can't do that correctly unless pytorch have the `silu_backward` aten symbol.

One thing I can do is to reuse `silu` symbol for backward. I think it will make `IR graph dump` incorrect but I think that should not impact the correctness of the computation. @bdhirsh wdyt?

**Comment by bdhirsh** [comment]
Hmm, https://github.com/pytorch/pytorch/pull/58661/ is the PR that ported `silu_backward` to structured in May, but it looks like before that PR, the out= variant (`silu_backward.grad_input`) didn't exist. I'm not sure at what point `silu_backward.grad_input` existed in a state that worked with pytorch/xla

There's an existing composite kernel for `silu_backward` though (the functional variant), so I think we probably just want to rewrite it so that there's also a composite kernel for the out= variant. I can add one.

**Comment by JackCaoG** [comment]
@bdhirsh Any chance this can be back ported to pytorch 1.10?

**Comment by bdhirsh** [comment]
I'm not 100% sure about being able to patch in a fix to 1.10, but I can check. It would probably easier if we can show that this bug didn't exist in 1.9. I'm still not sure how it could have worked before though, because I don't think the `out=` variant ever worked with pytorch/xla. Alternatively, if it's easier to patch into pytorch/xla then adding your lowering for `silu_backward` is probably fine (but... maybe not long-term what we want).

 @rwightman do you have a script to repro with, that worked in 1.9?

**Comment by rwightman** [comment]
@bdhirsh it also works fine in 1.8 FYI, I've been training with SiLU models on PyTorch XLA for better part of a year now.

I'm using train scripts from this branch in my model repo (https://github.com/rwightman/pytorch-image-models/tree/bits_and_tpu/timm/bits), but you should also be able to use any of the models with silu in your own scripts to verify ... (ie `timm.create_model('efficientnet_b0')`)

Last part of a model repr dump and type of the activations I just loaded in both 1.8 and 1.9 on a TPU w/ PyTorch XLA:

```
  (conv_head): Conv2d(320, 1280, kernel_size=(1, 1), stride=(1, 1), bias=False)
  (bn2): BatchNorm2d(1280, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
  (act2): SiLU(inplace=True)
  (global_pool): SelectAdaptivePool2d (pool_type=avg, flatten=Flatten(start_dim=1, end_dim=-1))
  (classifier): Linear(in_features=1280, out_features=1000, bias=True)
)
type: <class 'torch.nn.modules.activation.SiLU'>
```

Trains with no lowering issues in 1.8 or 1.9 at expected throughput.

**Comment by JackCaoG** [comment]
Brain and I agree that we will merge this temporary fix into master and 1.10 to unblock Ross. We will fix it properly(tracked in https://github.com/pytorch/xla/issues/3196) in the future.

**Comment by miaoshasha** [comment]
we updated the 1.10 1vm image tpu-vm-pt-1.10, silu works fine. Closing this bug.
